src/Engine/ProtoCore/ClassTable.cs

```diff
@@ -216,9 +216,22 @@ public ProcedureNode GetMemberFunction(string procName, List<ProtoCore.Type> arg
             if (ProcTable == null)
                 return null;
 
-            ProcedureNode procNode = null;
+            ProcedureMatchOptions opts = new ProcedureMatchOptions()
+            {
+                FunctionName = procName,
+                ParameterTypes = argTypeList,
+                ExcludeAutoGeneratedThisProc = true,
+                ExactMatchWithNumArgs = false,
+                ExactMatchWithArgTypes = false
```

Comment:
How did you arrive at these flag values?

Reply:
answered above